colcon-build-and-test/action.yaml

```diff
@@ -46,16 +46,23 @@ runs:
         DEBIAN_FRONTEND=noninteractive rosdep install -yqq --from-paths . --ignore-src --rosdistro ${{ inputs.rosdistro }}
       shell: bash
 
+    - name: Set colcon-mixin-repository
+      run: |
+        colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+        colcon mixin update default
+      shell: bash
+
     - name: Build
       run: |
         . /opt/ros/${{ inputs.rosdistro }}/setup.sh
         colcon build --event-handlers console_cohesion+ \
           --packages-above-and-dependencies ${{ inputs.target-packages }} \
           --cmake-args \
             -DCMAKE_BUILD_TYPE=Release \
-            -DCMAKE_C_FLAGS="--coverage -DCOVERAGE_RUN=1" \
-            -DCMAKE_CXX_FLAGS="--coverage -DCOVERAGE_RUN=1" \
-            -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+            -DCMAKE_C_FLAGS="-DCOVERAGE_RUN=1" \
+            -DCMAKE_CXX_FLAGS="-DCOVERAGE_RUN=1" \
```

Comment:
What's `DCOVERAGE_RUN` for?

Seeing this file, is it unnecessary? Or should we send a PR?
https://github.com/colcon/colcon-mixin-repository/blob/d770f1131ba61d81686c057e7af5d462523fb8a0/coverage.mixin#L5-L6

Reply:
https://github.com/colcon/colcon-mixin-repository/pull/11

Since it is non-standard, We should remove it.

Reply:
Changed 17d1ad9

Reply:
I see, thank you!